libpod/healthcheck_linux.go

```diff
@@ -152,10 +172,13 @@ func (c *Container) disableHealthCheckSystemd(isStartup bool) bool {
 }
 
 // Systemd unit name for the healthcheck systemd unit
```

Comment:
Add a comment what "bare" does because the name is not clear (I cannot think of a better one either)